examples/tutorials/DeepChem_AntibodyTutorial_Simplified.ipynb

```diff
@@ -0,0 +1,3696 @@
+{
```

Comment:
<p>Can have simple version as main and have the more advanced as a subsection marked "Advanced (can skip on first reading)" or something to that effect.</p><p> </p><p>Add definitions for terms like "somatic recombination" that a new reader may not know.</p><p> </p><p>Also please add explicit [1], [2] etc links to the reference in the ref section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4012/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>